packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -220,6 +254,25 @@ describe('entrypoints', () => {
           });
         });
 
+        it('run network-info exits with code 1 when cache duration is out of range', (done) => {
+          const cacheDurationOutOfRange = '3000';
```

Comment:
```suggestion
          expect.assertions(2);
          const cacheDurationOutOfRange = '3000';
```

Sorry, I should have been more explicit. Just ensure any of the callback-style tests with assertions in handlers follow this pattern (eg. line 277 as well)

Reply:
Does it mean to add assertions in all other tests that are missing?

Reply:
Yes. It's to ensure future regression causing the assertions to not be executed do not produce a false-positive.

Reply:
[00d4bba](https://github.com/input-output-hk/cardano-js-sdk/pull/276/commits/00d4bbae9dd517b46acd8abc7125bcff480b513c)